src/rules.ts

```
@@ -0,0 +1,42 @@
+
+export let getRules = () => {
+  let rulesData = new Uint32Array(2 ** 9);
+  for (let i = 0; i < 2 ** 9; i++) {
+    rulesData[i] = Math.random() < 0.8 ? 1 : 0;
+    // rulesData[i] = rules[1][i];
+  }
+
+  return rulesData;
+};
+
+export let getInitialCells = (width: number, height: number) => {
```

Biome

[error] This let declares a variable that is only assigned once.

'getInitialCells' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)